packages/base/src/commands.ts

```diff
@@ -971,6 +937,17 @@ export function addCommands(
         return;
       }
       current.wireframe = !current.wireframe;
+
+      const buttonNode = document.querySelector(
+        `[data-command="${CommandIDs.wireframe}"]`
+      );
```

Comment:
If I have multiple CAD documents opened. This selector will not select the correct one.

Reply:
Ahh yes, you're right